lib/dynamic.js

```diff
@@ -71,6 +71,13 @@ export default function dynamicComponent (p, o) {
           this.state.AsyncComponent = AsyncComponent
         }
       })
+        .catch(
+          this.ssr
+          ? undefined
```

Comment:
Won't this silently swallow the error when it happens server side? 🤔

Reply:
No, it would not make anything worse than it is. Both `.catch(cb?)` and `.then(cb?)` are actually a wrappers around `.then(successHandler?, errorHandler?)`. Let me explain it with shell example:

```
# jabher @ DESKTOP-DTTUH1G in /mnt/c/Users/jabher [1:05:39]
$ node
> process.on("unhandledRejection", console.warn.bind(null, 'in uncaught rejecton')); void 0
undefined
> Promise.reject('test').catch(undefined); void 0
undefined
> in uncaught rejecton test Promise {
  <rejected> 'test',
  domain:
   Domain {
     domain: null,
     _events: { error: [Function: debugDomainError] },
     _eventsCount: 1,
     _maxListeners: undefined,
     members: [] } }
```

Reply:
Oh, nice! 👌 So returning undefined actually will bubble it up 😄 TIL 👍